x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/distributed_lock_manager/distributed_lock_manager.spec.ts

```
@@ -645,15 +645,21 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
   });
 }
 
-function clearAllLocks(es: Client) {
-  return es.deleteByQuery(
-    {
-      index: LOCKS_CONCRETE_INDEX_NAME,
-      query: { match_all: {} },
-      refresh: true,
-    },
-    { ignore: [404] }
-  );
+function clearAllLocks(es: Client, log: ToolingLog) {
+  try {
+    return es.deleteByQuery(
```

Do we know what’s causing the flakiness? Would it make sense to use `retry` here?

---

Yes, it's the conflict I mentioned in the description.

---

In this case we don't need to retry. Ignoring the error (proceeding on conflicts) should be fine.